### PR TITLE
Create filter function

### PR DESCRIPTION
**Pull request by thinhbd-1256**
Filter function for website

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -3,11 +3,37 @@ class EventsController < ApplicationController
...
+
+    @events = @filterrific.find.page(params[:page]).per(12)
+
+    respond_to do |format|
+      format.html
+      format.js
+    end
+
+    rescue ActiveRecord::RecordNotFound => e
+      puts "Had to reset filterrific params: #{ e.message }"
```
Comment (annhoang191) [inline app/controllers/events_controller.rb]: #{e.message}

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -7,11 +7,43 @@
...
-        <%= render event %>
+  <div class="row">
+    <div class="col-lg-12 col-md-12">
+      <%= form_for_filterrific @filterrific do |f| %>
+        <div class="row">
+          <div class="col-lg-3 col-md-3">
+            <label>City</label>
+            <div class="dropdown-filter">
+              <%= f.select :with_city_id, @filterrific.select_options[:with_city_id],
+                {include_blank: "- Any -"} %>
```
Comment (annhoang191) [inline app/views/events/index.html.erb]: i18n

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -7,11 +7,43 @@
...
+            </div>
+          <div class="col-lg-3 col-md-3">
+            <label>Event Start date</label>
+            <div class="dropdown-filter">
+              <%= f.text_field :with_date_start_gte, class: "js-datepicker" %>
+            </div>
+          </div>
+        </div>
+        <div class="row reset-filter">
+          <%= link_to "Reset filters", reset_filterrific_url %>
```
Comment (annhoang191) [inline app/views/events/index.html.erb]: i18n

**Comment by thinhbd-1256** [comment]
ready

**Pull request closed without merging**